cpu/esp32/esp-now/esp_now_netdev.c

```diff
@@ -87,7 +87,7 @@ static inline int _get_mac_from_iid(uint8_t *iid, uint8_t *mac)
 static xtimer_t _esp_now_scan_peers_timer;
 static bool _esp_now_scan_peers_done = false;
 
-static bool _esp_now_add_peer (uint8_t* bssid, uint8_t channel, uint8_t* key)
+static bool _esp_now_add_peer(uint8_t* bssid, uint8_t channel, uint8_t* key)
```

Comment:
Spaces between function names and open parentheses are not explicitly forbidden. Depending on the editor used and syntax highlighting of the editor, the code may be more readable with a single space in between.
Anyway, I don't think that these changes were really necessary.

Reply:
https://github.com/RIOT-OS/RIOT/blob/7a5142f7bcaf2ec35de74774ea5b1beaeae64caa/cpu/esp32/esp-now/esp_now_netdev.c#L29 is not needed any longer.